visualize.py

```diff
@@ -10,7 +10,7 @@
 
 # Calculate alpha coefficients for the first test example
 with tf.Session() as sess:
-    saver.restore(sess, "model")
+    saver.restore(sess, "trained.model")
```

Comment:
Why did you change the name? And if it matters, shouldn't the name from line 9 be changed too? I would remain the old name for simplicity

Reply:
Actually it Raised a Error on the original code:
```
Traceback (most recent call last):
  File "train.py", line 154, in <module>
    saver.save(sess, "model")
  File "D:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\training\saver.py", line 1594, in save
    raise exc
ValueError: Parent directory of model doesn't exist, can't save.
```
While I run on Windows with MINGW64, maybe there doesn't exist a avaiable saving path.
I tried this line may solve it:
```
saver.save(sess, "./model")
```

Reply:
Wow, it's so surprisingly. You say  `saver.save(sess, "model")` doesn't work meanwhile `saver.save(sess, "trained.model")` works? Let's make it `saver.save(sess, "./model")` then if it works for Windows and you don't mind.